Kickstarter-iOS/Views/Controllers/ToggleViewController.swift

```diff
@@ -0,0 +1,62 @@
+import Library
+import Prelude
+import UIKit
+
+class ToggleViewController: UIViewController {
+  // MARK: - Properties
+
+  public lazy var titleLabel: UILabel = {
+    UILabel(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  public lazy var toggle: UISwitch = {
+    UISwitch(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  // MARK: - Lifecycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    _ = self.titleLabel
+      |> ksr_setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
```

Comment:
Makes sure that the `titleLabel` does not push the switch off screen and rather pushes its own text on next line.